### PR TITLE
use velocity_factor for the condition of stopping announce

### PR DESCRIPTION
**Pull request by JunichiMorimoto**
## PR Type

- Improvement

## Related Links

https://tier4.atlassian.net/browse/RT4-7571

## Description

use velocity_factor for the condition of stopping announce

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

## CI Checks

- **Build and test for PR**: Required to pass before the merge.

### CODE REVIEW

**Review by tkhmy — APPROVED** [review]
LGTM

File: src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py [hunk 1]
```diff
@@ -352,47 +372,28 @@ def stop_reason_checker_callback(self):
...
-                    dist_to_stop_pose = stop_factor.dist_to_stop_pose
-            if shortest_distance == -1 or shortest_distance > dist_to_stop_pose:
-                shortest_distance = dist_to_stop_pose
-                shortest_stop_reason = stop_reason.reason
+        if self._stop_announce_executed == True:
+            return
+
+        execute_stop_announce = False
+        for velocity_factor in self._autoware.information.velocity_factors:
+            self._node.get_logger().warning(
```
Comment (yabuta) [inline src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py:380]: この辺のwarning logは開発用ですかね？
それなら消したほうが良さそうかと

**Review by yabuta — CHANGES_REQUESTED** [review]

File: src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py (hunk 1 above)
Comment (JunichiMorimoto) [reply src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py]: 見逃してました、ありがとうございます
削除しました！

**Review by yabuta — APPROVED** [review]
LGTM

**Pull request merged**